hazelcast/connection.py

```diff
@@ -280,11 +280,18 @@ def _trigger_cluster_reconnection(self):
             self._start_connect_to_cluster_thread()
 
     def _init_wait_strategy(self, config):
+        cluster_connect_timeout = config.cluster_connect_timeout
+        if cluster_connect_timeout == -1:
+            # If the no timeout is specified by the
+            # user, or set to -1 explicitly, set
+            # the timeout to infinite.
+            cluster_connect_timeout = six.MAXSIZE
```

Comment:
As far as I've checked `sys.maxsize` is available on all versions of CPython. On Python 2.7 and Python 3.6 both evaluate `sys,maxsize` to `9223372036854775807`. So, I think it would be better to replace `sys.MAXSIZE` with `sys.maxsize`.